### DISCUSSION TITLE
Emissão de NF-e

### DISCUSSION THREAD

**Discussion by KelvinEger** [RamonSilva20/mapos#2249] [General]
Boa noite nobres!
Informo que estou implementando emissão de NF-e pelo Mapos!

Deixou meu e-mail para quem quiser entrar em contato: kelvin.eger@gmail.com

**Comment by KelvinEger** [comment]
Pessoal, estava com problemas no meu email! 
Respondi aos interessados em implementar NFE no MAPOS. 
Podem entrar em contato comigo no email acima que responderei!

**Reply by gleissonneves** [reply]
deixa o linkedin, @KelvinEger


abraço!

**Reply by cezargf** [reply]
Enviei um e-mail para você @KelvinEger . Tenho interesse.

**Comment by KelvinEger** [comment]
Funcionalidade da emissão de NF-e em desenvolvimento!

**Reply by gleissonneves** [reply]
fala, mans (@KelvinEger e @cezargf)! vamos criar um grupo no whatsApp? me procurem no linkedin @gleissonneves

**Reply by cezargf** [reply]
Oi @gleissonneves, vamos sim!

**Comment by GLEIDOSN** [comment]
Opa pessoal, tudo bem?
Trabalho com fiscal e achei o sistema muito interessante e "limpo" para implementações.

**Comment by Gabevan61** [comment]
Tenho interresse dessa opção como podemos fazer

**Comment by KelvinEger** [comment]
Olá!
Estamos agora, na criação do novo módulo de integração da NF-e com o
sistema!
Previsão de mais dois meses de trabalho (pois fizemos nas nossas horas
complementares).
Assim que tiver novidades entro em contato.

Em seg., 7 de out. de 2024 às 14:55, Gabevan61 ***@***.***>
escreveu:

**Reply by GLEIDOSN** [reply]
Opa Kelvin, qualquer coisa posso ajudar nas horas vagas!

**Reply by CarlosRezini** [reply]
@KelvinEger    Só com NF-e ou tmb com NFS-e??

**Comment by KelvinEger** [comment]
Bom dia, tudo bem?
A ideia é a implementação de ambos!

Em qui., 24 de out. de 2024 às 04:17, CarlosRezini ***@***.***>
escreveu:

**Reply by ideniltonfreitas** [reply]
Como anda a implementação?

**Reply by wgiacomelli** [reply]
Seria uma boa, se tiverem mais informações ou se puder contribuir em algo avisem pq isso ajudará demais.

**Comment by ideniltonfreitas** [comment]
E como tá essa integração no momento atual?

**Reply by sarkozin** [reply]
tenho integração com nfe.io

**Comment by MakerHR** [comment]
Está dando certo a implementação? caso precise de ajuda estou a disposição

**Comment by GhosT-Exe1** [comment]
Como está indo? é pago? queria implementar também

**Comment by KelvinEger** [comment]
Olá, boa tarde!
Estamos na reta final de testes, acredito que ainda este mês estará
disponível para quem desejar.
Teremos uma pequena taxa para ajudar nos custeios da equipe e dos materiais
necessários para o desenvolvimento.


Em dom., 5 de jan. de 2025 às 11:13, GHOST EXE!? ***@***.***>
escreveu:

**Reply by GhosT-Exe1** [reply]
Boa tarde!
Vocês tem algum grupo para informar quando estiver finalizado? e quanto será a taxa? estou interessado

**Comment by KelvinEger** [comment]
Estamos procurando um parceiro para homologar a implementação!
Chame no whatsapp 47991284658 para conversamos melhor!

Em dom., 5 de jan. de 2025 às 15:50, GHOST EXE!? ***@***.***>
escreveu:

**Reply by robsonhess** [reply]
olá, podemos fazer o teste, tenho certificado digital para homologação e estou interessado no sistema com nfe.

**Comment by luisguilhermew** [comment]
Temos mais novidades sobre NF-e?

**Comment by giampaulo** [comment]
Eai pessoal. Como está o andamento ?

**Comment by DataPowerTI** [comment]
Bom dia!
Estou implementeando agora a emissão de nf dentro do MapOS, sem software de terceiros, e sem mensalidade, esse que o amigo colocou que esta fazendo a integração custa mais de 1000,00 por ano...
Estou fazendo um aqui aberto, assim que der td certo aviso vcs.

**Reply by luisguilhermew** [reply]
Excelente!

**Reply by cezargf** [reply]
Conte comigo, caso precise de ajuda. Gostaria muito de ajudar se possível.

**Reply by tiagobeulke** [reply]
> Bom dia! Estou implementeando agora a emissão de nf dentro do MapOS, sem software de terceiros, e sem mensalidade, esse que o amigo colocou que esta fazendo a integração custa mais de 1000,00 por ano... Estou fazendo um aqui aberto, assim que der td certo aviso vcs.

alguma novidade deste sistema de NFe e  NFS-e

**Reply by alessandrodesign** [reply]
Recentemente fiz a implementação da NFSe Nacional em um projeto que trabalho, se precisar de alguma ajuda, avise. @KelvinEger

**Comment by jdcassol** [comment]
Boa Tarde, estou customizando meu Mapos e tenho interesse como proceder ?
